alacritty/src/event.rs

```diff
@@ -64,6 +65,386 @@ const MAX_SEARCH_HISTORY_SIZE: usize = 255;
+                | WindowEvent::TouchpadPressure { .. }
+                | WindowEvent::RotationGesture { .. }
+                | WindowEvent::CursorEntered { .. }
+                | WindowEvent::PinchGesture { .. }
+                | WindowEvent::AxisMotion { .. }
+                | WindowEvent::PanGesture { .. }
+                | WindowEvent::HoveredFileCancelled
+                | WindowEvent::Destroyed
+                | WindowEvent::ThemeChanged(_)
+                | WindowEvent::HoveredFile(_)
+                | WindowEvent::Moved(_)
+        )
+    }
+}
+
+impl ApplicationHandler<Event> for Processor {
+    // The event loop just got initialized. Create a window.
+    fn resumed(&mut self, event_loop: &ActiveEventLoop) {
+        // Creating window inside event loop is required for platforms like macOS to
+        // properly initialize state, like tab management. Otherwise the first
+        // window won't handle tabs.
+        let initial_window_options = match self.initial_window_options.take() {
+            Some(initial_window_options) => initial_window_options,
+            None => return,
+        };
+
+        if let Err(err) = self.create_initial_window(event_loop, initial_window_options) {
+            self.initial_window_error = Some(err);
+            event_loop.exit();
+            return;
+        }
+
+        info!("Initialisation complete");
+    }
+
+    fn window_event(
+        &mut self,
+        _event_loop: &ActiveEventLoop,
+        window_id: WindowId,
+        event: WindowEvent,
+    ) {
+        if self.config.debug.print_events {
+            info!("winit event: {:?}", event);
+        }
+
+        // Ignore all events we do not care about.
+        if Self::skip_window_event(&event) {
+            return;
+        }
+
+        let window_context = match self.windows.get_mut(&window_id) {
+            Some(window_context) => window_context,
+            None => return,
+        };
+
+        match event {
+            WindowEvent::RedrawRequested => {
+                window_context.handle_event(
+                    #[cfg(target_os = "macos")]
+                    _event_loop,
+                    &self.proxy,
+                    &mut self.clipboard,
+                    &mut self.scheduler,
+                    WinitEvent::WindowEvent { window_id, event },
+                );
+
+                window_context.draw(&mut self.scheduler);
+            },
+            event => {
+                window_context.handle_event(
+                    #[cfg(target_os = "macos")]
+                    _event_loop,
+                    &self.proxy,
+                    &mut self.clipboard,
+                    &mut self.scheduler,
+                    WinitEvent::WindowEvent { window_id, event },
+                );
+            },
+        }
+    }
+
+    fn new_events(&mut self, event_loop: &ActiveEventLoop, cause: StartCause) {
+        let _ = (event_loop, cause);
```

Comment:
What exactly is this for? It does nothing from what I can tell.

Reply:
Yeah, I forgot to remove.